examples/jsm/interactive/HTMLMesh.js

```diff
@@ -25,9 +25,9 @@ class HTMLMesh extends Mesh {
 
 		}
 
-		this.addEventListener( 'mousedown', onEvent );
-		this.addEventListener( 'mousemove', onEvent );
-		this.addEventListener( 'mouseup', onEvent );
+		this.addEventListener( 'pointerdown', onEvent );
+		this.addEventListener( 'pointermove', onEvent );
+		this.addEventListener( 'pointerup', onEvent );
 		this.addEventListener( 'click', onEvent );
```

Comment:
Does it makes sense to have event listeners for `pointerdown` and `click`? Isn't that redundant?

Reply:
I'm just asking since we are only using pointer events in controls classes and example code. If we keep the `click` event listener, there must be a compelling reason.

Reply:
I agree that `click` is a bit odd as it implies a using a mouse... The distinction between `select` and `selectend` events seems to be reserved for succesful end of a primary action. In that case it might be more analogous to `pointerend'?
https://developer.mozilla.org/en-US/docs/Web/API/WebXR_Device_API/Inputs#actions